README.asciidoc

```diff
@@ -1629,17 +1629,30 @@ This param is in development and may change in the future.
 [[new-feature]]
 === New experimental feature
 
-experimental::[]
+Experimental language is deprecated.
 
-experimental::[https://github.com/elastic/docs/issues/505]
+We decided on the much less raw sounding "technical preview".
 
-experimental::[{issue}505]
+See below.
```

Comment:
I think we need to remove the starting code block on lines 1627-1628. As is, this renders oddly:

<img width="1081" alt="Screen Shot 2022-01-06 at 4 24 12 PM" src="https://user-images.githubusercontent.com/40268737/148454251-bb436c05-17c1-4ec0-8d23-a9710309d56d.PNG">

Reply:
Thanks @jrodewig -- better now?

Reply:
Perfect. Thanks!